PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+    document.add_paragraph()
+
+    if ktoryoddzial.get() == '50':
+        przygzatw = document.add_paragraph('Przygotował:\t' + przygotowal + '\tZatwierdził:\t' + zatwierdzil)
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(13.75))
+
+        datydaty = document.add_paragraph('Data:\t' + kiedy_p + '\tData:\t' + kiedy_z)
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(13.75))
+
+        podpispodpis = document.add_paragraph('Podpis:\tPodpis:')
+        podpispodpis.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+        podpispodpis.paragraph_format.tab_stops.add_tab_stop(Cm(11.5))
+    else:
+        przygzatw = document.add_paragraph('Przygotował:\t' + przygotowal)
+        przygzatw.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+
+        datydaty = document.add_paragraph('Data:\t' + kiedy_p)
+        datydaty.paragraph_format.tab_stops.add_tab_stop(Cm(2.5))
+
+        podpispodpis = document.add_paragraph('Podpis:')
+
+    def filesave():
+        if ktoryoddzial.get() == '50':
+            sss = nQC
+        else:
+            sss = 'QC-'+ktoryoddzial.get()+'-'+nQC+'-'+strftime('%y')
+        filename = filedialog.asksaveasfilename(defaultextension='.docx', initialfile=sss)
+        if not filename: return
+        document.save(filename)
+        pytanieootwarcie = messagebox.askyesno("Co teraz?", "Gratuluję sukcesu!\nCzy chcesz teraz otworzyć zapisany dokument?", default="yes")
+        if pytanieootwarcie:
+            startfile(filename)
+        else:
+            pass
+    filesave()
+
+def generujpoangielsku():
+    wyplujdane()
+    generatortolerancji()
+    global checkIle, checkIle2
+    nQC = numerQC.get()
+    nKRP = numerZP.get()
+    nTU = tu.get()
+    ile = int(ilosc.get())
+    rodzdlu = waz.get()
+    dlu = dlugosc.get()
+    med = medium.get()
+    cisR = cisnienieRob.get()
+    pdcsn = podcisn.get().replace('.',',')
+    tw = tempWew.get()
+    tz = tempZew.get()
+    medB = mediumBadania.get()
+    cisT = cisnienieTest.get()
+    cz = czasTestu.get()
+    zam = numerZam.get()
+    medben = ''
+    hashave = 'water tested'
+    przygotowal = entryprzygotowal.get()
+    kiedy_p = entrykiedyprzygotowal.get()
+    zatwierdzil = entryzatwierdził.get()
+    kiedy_z = entrykiedyzatwierdzil.get()
+    kli1 = entryKlient1.get()
+    kli2 = entryKlient2.get()
+    kli3 = entryKlient3.get()
+    konc1 = koncowka1entry.get()
+    konc2 = koncowka2entry.get()
+    uwagi = poleuwagi.get(0.0, "end-1c")
+    kol1 = listaparametrow[0]
+    kol2 = listaparametrow[1]
+    kol3 = listaparametrow[2]
+    kol4 = listaparametrow[3]
+    kol5 = listaparametrow[4]
+    kol6 = listap
```

Comment:
Generalnie powinno się unikać takich porównań z liczbami. Liczba na pewno coś znaczy, ale nie wiadomo co.  https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad